src/main/java/Jeff.java

```diff
@@ -0,0 +1,189 @@
+import java.util.Scanner;
+import java.util.Random;
+
+public class Jeff {
+
+    //Constants
+    public static final String DIVIDER = "____________________________________________________________";
+    public static final String  introText = """
+                ____________________________________________________________
+                Hello! I'm JEFF!!!
+                
+                Here are the type of tasks I can track!!!
+                'todo [description]'
+                'deadline [description] /by [some date]'
+                'event [description] /from [some date] /to [some date]'
+                
+                List of commands I support!!!!
+                Type 'list' to display everything you've said!
+                Type 'mark'/'unmark' to change the status of inputted tasks!
+                Type 'bye' to exit!
+                
+                If you give me nonsense, I will echo whatever you say!
+                Buuuuttt with random capitalisation!
+                ____________________________________________________________
+                """;
+
+    public static final String exitText = """
+                ____________________________________________________________
+                Bye. Hope to see you again soon!
+                ____________________________________________________________
+                """;
+    public static final int TODO_LENGTH = 5;
```

Comment:
I like how all these constants are defined, but perhaps they could be placed at the start of the class, or maybe just leave a space after the previous method to improve readability